### PR TITLE
Add plan custom fields

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

**Comment by dara123** [comment]
devs please review

File: spec/paths/custom-fields@{resource}@{name}.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ parameters:
       - "websites"
       - "contacts"
       - "products"
+      - "plans"
```
Comment (slavcodev) [inline spec/paths/custom-fields@{resource}@{name}.yaml]: Please segregate this definition to separate file, in order to reuse and do not duplicate this enum many and many times.

File: spec/paths/custom-fields@{resource}.yaml [hunk 2]
```diff
@@ -6,13 +6,7 @@ parameters:
...
     required: true
     enum:
-      - "customers"
-      - "payment-cards"
-      - "subscriptions"
-      - "transactions"
-      - "websites"
-      - "contacts"
-      - "products"
+      $ref: "#/definitions/CustomFieldResources"
```
Comment (slavcodev) [inline spec/paths/custom-fields@{resource}.yaml]: This is incorrect, please check https://github.com/Rebilly/RebillyAPI/blob/master/spec/swagger.yaml#L306

File: spec/paths/custom-fields@{resource}.yaml [hunk 3]
```diff
@@ -1,18 +1,6 @@
...
-    required: true
-    enum:
-      - "customers"
-      - "payment-cards"
-      - "subscriptions"
-      - "transactions"
-      - "websites"
-      - "contacts"
-      - "products"
+    $ref: "#/definitions/CustomFieldResources"
```
Comment (slavcodev) [inline spec/paths/custom-fields@{resource}.yaml]: please add the definition to `#/parameters` (sorry I've confused you with "separated file", I meant separate parameter definition in swagger.yaml)

File: spec/swagger.yaml [hunk 4]
```diff
@@ -369,6 +369,21 @@ parameters:
       - application/pdf
     default: application/json
     description: The response media type
+  customFieldResources:
```
Comment (slavcodev) [inline spec/swagger.yaml]: Typo: `customFieldResource`

**Pull request merged**